auslib/dockerflow.py

```diff
@@ -26,8 +28,12 @@ def heartbeat():
         """Per the Dockerflow spec:
         Respond to /__heartbeat__ with a HTTP 200 or 5xx on error. This should
         depend on services like the database to also ensure they are healthy."""
-        database_entry_value = heartbeat_database_fn(dbo)
-        return Response(str(database_entry_value), headers={"Cache-Control": "no-cache"})
+        try:
+            database_entry_value = heartbeat_database_fn(dbo)
+            return Response(str(database_entry_value), headers={"Cache-Control": "no-cache"})
+        except Exception as e:
+            logging.exception("Error occured: %s", e.message)
+            return Response(status=502, response=e.message)
```

Comment:
@mostlygeek - do you have any strong feelings about including the original Exception message? In local testing, this generates errors like this if I use the wrong db credentials:
> (OperationalError) (1045, "Access denied for user 'balrogadmin'@'172.17.0.4' (using password: YES)")

If this were to happen with the public app in production, the database server IP would be publicly available. I'm not sure how else to provide useful error messages for exceptions, though.